Lesson2Xcode/ComponentsViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ComponentsViewController.swift
+//  Lesson2Xcode
+//
+//  Created by Arslan Rashidov on 20.09.2021.
+//
+
+import Foundation
+import UIKit
+
+class ComponentsViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
+    
+    @IBOutlet weak var PickerViewComponent: UIPickerView!
+    var pickerData = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6"]
+    
+    @IBOutlet weak var SegmentedControlComponent: UISegmentedControl!
+    @IBOutlet weak var SwitchComponent: UISwitch!
+    @IBOutlet weak var SliderComponent: UISlider!
+    @IBOutlet weak var StepperComponent: UIStepper!
+    @IBOutlet weak var ButtonComponent: UIButton!
+    @IBOutlet weak var ProgressViewComponent: UIProgressView!
+    @IBOutlet weak var LabelComponent: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+    
+    @IBAction func ChangeValuesButton(_ sender: Any) {
```

Comment:
```suggestion
    @IBAction func changeValuesButton(_ sender: Any) {
```
везде исправляй названия